src/go/options/configgenerator.go

```diff
@@ -108,8 +108,11 @@ type ConfigGeneratorOptions struct {
 	ConnectionBufferLimitBytes    int
 
 	// JwtAuthn related flags
-	DisableJwksAsyncFetch bool
-	JwksCacheDurationInS  int
+	DisableJwksAsyncFetch             bool
```

Comment:
Diff looks weird here, is there a change in whitespace? Can you run `make format`?

Reply:
make format generates identical code.
I think the go formatting takes this line and the next four and decides to align based on the widest variable name
`JwksFetchRetryBackOffBaseInterval` three lines below